### ISSUE TITLE
AV: Double tap on mobile doesn't move to full screen

### ISSUE THREAD

**Issue by lizcameron** [bbc/simorgh#4655]
**Describe the bug**
Double tapping the player and clicking the full screen button on iOS causes the player window to go blank. The audio continues to play. This can be manually resolved by refreshing the page or rotating the screen, after which the video returns to the window.

**To Reproduce**
Steps to reproduce the behaviour:

1. On an iOS device, go to https://www.test.bbc.co.uk/news/articles/cn7k01xp8kxo
2. Scroll down to the video and click play
3. Double tap on the video and/or click the fullscreen button
4. See error: video player goes blank but audio continues to play. Full screen is not enabled.

**Expected behavior**
Double tap should not cause the video to go blank. Instead it should cause the video to move to full screen.

**Screenshots**

**Smartphone (please complete the following information):**

- Device: iPhone 8 
- OS: OS13.1.3
- Browser: Safari & Chrome browsers

**Testing notes**
Note that this can't be replicated on Pixel 2 in Chrome - see notes from cross-browser testing to see more details of devices/OS this has been tested on.

**Additional context**

**Comment by HarveyPeachey** [comment]
After taking a look at the SMP confluence docs and a discussion with the media playout team, our implementation of our AV player is not going to allow us to fix the fullscreen bugs for iOS without causing further issues:

> iOS pre 12.2 and on all iPhones does not support the fullscreen API, this means SMP cannot provide the "fake fullscreen" behaviour it does on these because it only works on the document that it is embedded into - ie it maximises within the iframe, not the whole document. (you could listen to the before and after fullscreen events and resize your iframe too, but that is likely to be problematical due to the way iOS manages resize events in iframes too).

> Safari doesn't fire enough resize events or orientationchange events, so the player can find itself thinking it's larger or smaller the fake window. You end up with loads of `setTimeouts` and guessing the changes etc.

What media playout recommends is disabling full-screen altogether for the devices that won't work. 

I've tested the solution anyway of resizing the iframe and it seems to look okay for on an iPhone on iOS 13.1.3 using this styling: https://gist.github.com/HarveyPeachey/9b78c05278f50b75510d5c51b5c183bb. But the media playout team said that iOS 11 and higher might be okay, but anything below is where the problems lie, and our minimum support for Simorgh is iOS 10.

In summary, we can't fix this issue reliably and have been advised by the media playout team not to try the only workaround. Going forward we need to discuss what our options could be, such as disabling fullscreen mode for iOS users.

**Comment by lizcameron** [comment]
Thanks for doing this investigation @HarveyPeachey 
I think the next step is to investigate whether full-screen can be disabled just for iOS

**Comment by HarveyPeachey** [comment]
@lizcameron Do you want me to create an investigation issue for that? If so I can close this investigation at the same time.

**Comment by simonsinclair** [comment]
Thanks for investigating this, @HarveyPeachey.

> the media playout team said that iOS 11 and higher might be okay, but anything below is where the problems lie

I notice they say anything above iOS 11 _**might**_ be ok - would it be a good idea confirm this before considering our options? iOS 13 is now current and I imagine it'd be natural for Simorgh's iOS support version to increase also(?).

**Comment by rhenshaw56** [comment]
Thanks for doing this @HarveyPeachey, I think if we're able to find a way to disable for just IOS and not other devices, it would be fine as well.

**Comment by aeroplanejane** [comment]
@lizcameron Has this been disabled by default for all IOS devices?

**Comment by lizcameron** [comment]
@aeroplanejane it's disabled by default for _most_ IOS devices. We're using the `insideIframe` setting for SMP which disables "Fake Fullscreen". This will affect some quite old android devices and most iPhone OS. It does still work on iPad, though.
